### PR TITLE
devops 3.0 roadmap

### PR DESCRIPTION
**Pull request by runzexia**
/kind docs
/area devops
/milestone 3.0.0
/assign
/cc @FeynmanZhou

### CODE REVIEW

File: docs/roadmap.md [hunk 1]
```diff
@@ -30,6 +30,12 @@ KubeSphere Roadmap demonstrates a list of open source product development plans
 
 #### DevOps
 
+- [ ] Create / Edit Pipeline Process Optimization.
```
Comment (FeynmanZhou) [inline docs/roadmap.md]: "Create / Edit Pipeline Process Optimization." is too vague, can you give much detailed description?  @runzexia

File: docs/roadmap.md [hunk 2]
```diff
@@ -30,6 +30,12 @@ KubeSphere Roadmap demonstrates a list of open source product development plans
...
+- [ ] S2I/B2I supports webhook.
+- [ ] Image registry optimization.
```
Comment (FeynmanZhou) [inline docs/roadmap.md]: Image registry management optimization, provide control panel for private image registry (e.g. manage images within Harbor)

**Comment by FeynmanZhou** [comment]
I don't see any updates after review and comments. Maybe we can merge it then go to refine this part.

**Comment by shaowenchen** [comment]
> I don't see any updates after review and comments. Maybe we can merge it then go to refine this part.

Yes，please merge it and I will repair it.

By the way，all PR labeled by `lgtm` need quickly merge to Integrated branch. Usually, it's master branch. `lgtm` means I'm responsible for PR.

We can control the new function through the switch.

**Pull request merged**